web/controllers/account_controller.ex

```diff
@@ -1,14 +1,51 @@
 defmodule Entice.Web.AccountController do
   use Entice.Web.Web, :controller
   alias Entice.Web.Account
+  alias Entice.Web.Invitation
+  alias Entice.Web.Queries
+  require Logger
+
+  plug :ensure_login when action in [:request_invite]
 
   def create(conn, _params) do
     email = conn.params["email"]
     password = conn.params["password"]
+    invite_key = conn.params["inviteKey"]
```

Comment:
Not entirely sure, but I think the convention for this parameter is snake_case rather than camelCase.... can you check pls? (So ```invite_key``` rather than ```inviteKey```)